src/builders/nodejs/granular/default.nix

```diff
@@ -148,51 +151,51 @@ let
   '';
 
   # Generates a derivation for a specific package name + version
-  makePackage = name: version:
-    let
-
-      deps = getDependencies name version;
-
-      nodeDeps =
-        lib.forEach
-          deps
-          (dep: packages."${dep.name}"."${dep.version}" );
-
-      dependenciesJson = b.toJSON
-        (lib.listToAttrs
-          (b.map
-            (dep: lib.nameValuePair dep.name dep.version)
-            deps));
-
-      electronDep =
-        if ! isMainPackage name version then
-          null
-        else
-          lib.findFirst
-            (dep: dep.name == "electron")
-            null
-            deps;
-
-      electronVersionMajor =
-        lib.versions.major electronDep.version;
-
-      electronHeaders =
-        if electronDep == null then
-          null
-        else
-          pkgs."electron_${electronVersionMajor}".headers;
-
-
-      pkg =
-        produceDerivation name (stdenv.mkDerivation rec {
-
+  makePackage = name: version: let
+    deps = getDependencies name version;
+
```

Comment:
...and here it is pulled up again

Reply:
Not a big issue, just noticing the inconsistency. Maybe it's for a good reason, maybe not...

Reply:
Thanks for noticing, this kind of feedback is very useful!